File: modul 4/PRAK404-2410817320004-AmandhaCitraMustika.c
#include <stdio.h>
int main(){
int pilihan;
float pertama,kedua,hasil;

while (1){
printf("1. Penjumlahan\n");
printf("2. Pengurangan\n");
printf("3. Perkalian\n");
printf("4. Pembagian\n");
printf("5. Exit\n");
printf("Masukkan Pilihan :");
scanf("%d",&pilihan);

if(pilihan >=1 && pilihan <=4){
printf("Masukkan nilai Pertama :");
scanf("%f",&pertama);
printf("Masukkan nilai kedua :");
scanf("%f",&kedua);

switch(pilihan){
    case 1:
    hasil=pertama+kedua;
    printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f \n\n", pertama,kedua,hasil);
    break;

    case 2:
    hasil=pertama-kedua;
    printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f \n\n",pertama,kedua,hasil);
    break;

    case 3:
    hasil=pertama*kedua;
    printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f \n\n",pertama,kedua,hasil);
    break;

    case 4:
        if(kedua!=0 || pertama!=0){
    hasil=pertama/kedua;
    printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f \n\n",pertama,kedua,hasil);
    } else{
        printf("pembagian nol tidak bisa\n\n");}
    break; 
    } 
    }

    else if(pilihan==5){
        printf("Terimakasih, telah menggunakan kalkulator AMANDHA CITRA MUSTIKA\n\n"); 
        break;}


    else {printf("Input anda salah, silahkan coba lagi \n\n");}

}
    return 0;
}

File: modul 4/PRAK404-2410817320004-AmandhaCitraMustika.py
while True:
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    pilihan=int(input("Masukkan Pilihan :"))

    if 1<=pilihan<=4:
        pertama=float(input("Masukkan nilai pertama :"))
        kedua=float(input("Masukkan nilai kedua:"))

        if pilihan==1 :
            hasil=pertama+kedua
            print(f"Hasil penjumlahan antara {pertama:.2f} dengan {kedua:.2f} adalah {hasil:.2f}\n\n")
        elif pilihan==2 :
            hasil=pertama-kedua
            print(f"Hasil pengurangan antara {pertama:.2f} dengan {kedua:.2f} adalah {hasil:.2f}\n\n")
        elif pilihan==3 :
            hasil=pertama*kedua
            print(f"Hasil perkalian antara {pertama:.2f} dengan {kedua:.2f} adalah {hasil:.2f}\n\n")
        elif pilihan==4 :
            if pertama!=0 | kedua!=0 :
                hasil=pertama/kedua
                print(f"Hasil pembagian antara {pertama:.2f} dengan {kedua:.2f} adalah {hasil:.2f}\n\n")
            else : print("Pembagian dengan nol tidak bisa\n\n")

    elif pilihan==5 :
            print("Terima kasih, telah menggunakan kalkulator AMANDHA CITRA MUSTIKA\n\n")
            break
    else :
            print("Input anda salah, silahkan coba lagi\n\n")